vignettes/how-to/communicate-between-modules.Rmd

```diff
@@ -0,0 +1,136 @@
+---
+title: "How-to: Communicate between modules"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{How-to: Communicate between modules}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+State management is a topic that comes along very often when it comes to reactive programming. 
+
+As the application grows in size and becomes more intricate, it's common to find an increasing number of 
+[Shiny modules](https://shiny.posit.co/r/articles/improve/modules/) distributed across various levels of depth.
+This results in the necessity to share information, particularly the application's state, among these diverse Shiny modules.
+
+While there are many ways to approach this problem here (Figure 1) we are going to present one that consists in passing reactive 
+values between two sibling modules.
+
+Suppose we have a data processing module and a plotting module, both functioning as siblings, with their respective roles
+being to process and exhibit data in a given plot. These sibling modules are nested within another module, which we've
+denoted as `main.R` which is our main module within a Rhino app.
+
+![Figure 1. Diagram illustrating how a reactive value is passed between sibling modules](images/communicate_between_modules_1.png)
+
+Let's explore how the code of these modules should look like:
+
+```R
```

Comment:
For consistency, please add `box` imports.

Reply:
done